web-registry/src/pages/ProjectReview/ProjectReview.tsx

```diff
@@ -157,10 +157,10 @@ export const ProjectReview: React.FC<React.PropsWithChildren<unknown>> = () => {
         onEditClick={() => navigate(`${editPath}/media`)}
       >
         {metadata?.['regen:previewPhoto'] && (
-          <Photo src={metadata?.['regen:previewPhoto']} />
+          <Photo src={metadata?.['regen:previewPhoto']['schema:url']} />
         )}
         {metadata?.['regen:galleryPhotos']?.map(
-          photo => photo && <Photo src={photo} />,
+          photo => photo && <Photo src={photo['schema:url']} />,
         )}
         {/* TODO: display story image or video https://github.com/regen-network/regen-registry/issues/1615 */}
```

Comment:
Well, this was not part of the title issue so I assume this would be done separately (I did not take it into account for the estimation too). The PR already is already quite big anyways so might be worth doing separately.

Reply:
as discussed this will be addressed in a separate issue